front/package-lock.json

```diff
@@ -0,0 +1,14811 @@
+{
```

Comment:
We currently have a yarn lock. Should we remove it and switch to npm for dependencies, or keep yarn and remove the npm lock file?

Reply:
As you prefer,  I'm not sure what the best option is.